src/Compilers/CSharp/Portable/Binder/Binder_Lookup.cs

```diff
@@ -1097,6 +1097,12 @@ internal SingleLookupResult CheckViability(Symbol symbol, int arity, LookupOptio
             {
                 return LookupResult.NotInvocable(unwrappedSymbol, symbol, diagnose);
             }
+            else if (InCref && !this.IsCrefAccessible(unwrappedSymbol))
+            {
+                var unwrappedSymbols = ImmutableArray.Create<Symbol>(unwrappedSymbol);
```

Comment:
I suppose it doesn't really matter, but we don't appear to use this if diagnose is false.  Maybe just inline the expression?